src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -345,10 +347,6 @@ public void motorConfigFalcon() {
 
     /** Zero all sensors, both Talons and Pigeon */
     public void zeroSensors() {
-      //BD  SupplyCurrentLimitConfiguration changeLimit = new SupplyCurrentLimitConfiguration(Constants.enable,
```

Comment:
this needs to be added back for drivers to actually change from dashboard during testing